pkgdb2/templates/package_request.html

```diff
@@ -17,7 +17,8 @@
     {{ render_field_in_row(form.description) }}
     {{ render_field_in_row(form.branches) }}
     {{ render_field_in_row(form.review_url, title="URL pointing directly to the
- location where the package review happened") }}
+ location where the package review happened",
+     placeholder="https://bugzilla.redhat.com/1234") }}
```

Comment:
@puiterwijk had a similar remarks, but the version here is also valid (and nicer imho).